website_snippet_country_dropdown/demo/assets.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
-     © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+<!-- (C) 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
```

Comment:
Use `Copyright` full word instead of abbreviations

Reply:
Done